src/Compilers/CSharp/Test/Semantic/Semantics/UsingStatementTests.cs

```diff
@@ -1317,9 +1317,13 @@ static void Main()
 
             Assert.Equal("System.IO.StreamWriter a", firstDeclaredSymbol.ToTestDisplayString());
 
+            var secondDeclaredSymbol = model.GetDeclaredSymbol(usingStatement.Declaration.Variables.Last());
+
+            Assert.Equal("System.IO.StreamReader b", secondDeclaredSymbol.ToTestDisplayString());
+
             var typeInfo = model.GetSymbolInfo(usingStatement.Declaration.Type);
-            // lowest/last bound node with associated syntax is being picked up. Fine for now.
```

Comment:
>// lowest/last bound node with associated syntax is being picked up. Fine for now. [](start = 12, length = 82)

Perhaps the comment should be adjusted rather than removed. Something like: "the type info uses the type inferred for the first declared local".